colly.go

```diff
@@ -267,6 +268,11 @@ func (c *Collector) DisableCookies() {
 	c.backend.Client.Jar = nil
 }
 
+// SetClientTimeout overrides the default timeout for this collector
```

Comment:
Could you extend this documentation string with the default timeout value?

Reply:
Yes! Good idea!